schema/image_test.go

```diff
@@ -37,3 +37,12 @@ func Test_BuildImageName_BranchAndSHAFormat(t *testing.T) {
 		t.Errorf("BuildImageName want: \"%s\", got: \"%s\"", want, got)
 	}
 }
+
+func Test_BuildImageName_RegistryWithPort(t *testing.T) {
+	want := "registry.domain:8080/image:latest"
+	got := BuildImageName(DefaultFormat, "registry.domain:8080/image", "ef384", "master")
```

Comment:
I think it is probably a good idea to also include one case with a port _and_ a tag, e.g. `registry.domain:8080/image:foo`

Reply:
I pushed a new version with that. Thank you for the hint.